detections/endpoint/crowdstrike_event_streams_event.yml

```diff
@@ -0,0 +1,69 @@
+name: CrowdStrike Event Stream Events
```

Comment:
Can we name this detection search better - catering to what the detection is trying to do?